modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/soy/UserCardTag.java

```diff
@@ -20,7 +20,10 @@
 
 /**
  * @author Julien Castelain
+ * @deprecated As of Cavanaugh (7.4.x), replaced by {@link
+ *             com.liferay.frontend.taglib.clay.servlet.taglib.UserCardTag}
  */
+@Deprecated
```

Comment:
I think we can simply delete this class now.

This would be our standard approach, but this case is slightly different.
- By keeping it we only maintain binary compatibility because backwards compatibility is ensured by the implementation of the new class. That is, only people directly extending `UserCardTag` in Java code will be affected. This is a very unlikely scenario.
- We'll be removing the classes that we deprecated in `7.3` so we'll already introduce a binary breaking change.
- We're removing `soy` completely in `7.4`, so even if the `extend` mechanism worked, the class is useless and won't do any good.

Reply:
👍

Reply:
If you want [this](https://github.com/liferay-frontend/liferay-portal/pull/434#discussion_r511868931) to be done in a different pr we can't remove tag yet.

Reply:
Yeah, there's no rush, just wanted to write it down. We an do it in the final tasks when we remove the temporary and other tags.